mle/agents/summarizer.py

```diff
@@ -7,18 +7,20 @@
 
 class SummaryAgent:
 
-    def __init__(self, model, github_repo: str, github_token: str = None, console=None):
+    def __init__(self, model, github_repo: str, username: str, github_token: str = None, console=None):
         """
         SummaryAgent: summary the workspace provided by the user.
 
         Args:
             model: the model to use.
+            username: the Github username of the user.
```

Comment:
so this is just for GitHub summary?

Reply:
why not call this SummaryGitHub

then we can have `SummaryZoom`

Reply:
so far, yes